opentech/apply/projects/templates/application_projects/project_detail.html

```diff
@@ -166,11 +166,26 @@ <h4 class="modal__header-bar">Request Payment</h4>
 
                 </div>
 
+                {% if object.is_in_progress %}
+                <div class="sidebar__inner">
+                    <h5>Supporting Documents</h5>
+
+                    <p><a href="{{ object.submission.get_absolute_url }}">Proposal</a></p>
+
+                    {% if request.user.is_apply_staff %}
+                    <p><a href="#">Approval form</a></p>
+                    <p><a href="#">Supporting documents</a></p>
+                    {% endif %}
+
+                    <p><a href="{{ object.contracts.first.file.url }}">Draft contract</a></p>
+                </div>
+                {% endif %}
+
                 {% if request.user.is_apply_staff and approvals %}
                 {% block project_approvals %}{% endblock %}
                 {% endif %}
 
-                {% if contracts %}
+                {% if not object.is_in_progress and contracts %}
```

Comment:
We should be showing the contract when they exist, they are relevant at all times.

Reply:
Ah, I misunderstood this line in the ticket:

> Applicant can only see proposal and contracts

Fixed and rebased.